### DISCUSSION TITLE
JSON Schema Docusaurus integration may be helpful for OpenAPI?

### DISCUSSION THREAD

**Discussion by jasikpark** [PaloAltoNetworks/docusaurus-openapi-docs#474] [General]
https://jy95.github.io/docusaurus-json-schema-plugin/ was released (found it released in https://join.slack.com/t/json-schema/shared_invite/zt-1q6z0hqdz-lY~gapEt0o2CsPw7wZIyXw)

I wonder if the UI could be of any help w/ displaying JSON responses? Probably not, but just wanted to put it on y'all's radar!

**Comment by sserrata** [comment]
Thanks for sharing @jasikpark, it looks interesting, especially since it appears to have complete coverage.

**Comment by jasikpark** [comment]
<img width="706" alt="image" src="https://user-images.githubusercontent.com/10626596/222826061-79ef5e79-1352-490d-8600-8497896a1c63.png">
https://jy95.github.io/docusaurus-json-schema-plugin/docs/faq

another fun moment for OpenAPI 3.1+
